core/src/main/java/org/ehcache/core/spi/store/Store.java

```diff
@@ -540,7 +540,7 @@
      * @param serviceConfigs the configurations the Provider may need to configure the Store
      * @return the Store honoring the configurations passed in
      */
-    <K, V> Store<K, V> createStore(Configuration<K, V> storeConfig, ServiceConfiguration<?>... serviceConfigs);
+    <K, V> Store<K, V> createStore(boolean useLoaderInAtomics, Configuration<K, V> storeConfig, ServiceConfiguration<?>... serviceConfigs);
```

Comment:
If we need to pass `useLoaderInAtomics` here then we should just fold it in to the `Store.Configuration` interface.

Reply:
I did think of it, but `useLoaderInAtomics` is a CacheManager level config, that's why it felt wrong to add it to the `Store`'s config. Its set at CacheManager level.

Reply:
It may only currently be settable at a CacheManager scope but it's not CacheManager config... it modifies the behavior of the Store, therefore it's store config.

Reply:
ok fair enough